frontend/src/hooks/useInventory.ts

```diff
@@ -0,0 +1,37 @@
+import {useEffect, useState} from "react";
+import axios from "axios";
+import {Inventory} from "../types/Inventory.ts";
+
+export default function useInventory() {
+    const [inventory, setInventory] = useState<Inventory>();
+    const playerId = "111";
+
+    function fetchInventory() {
+        axios.get('/api/inventory/player/' + playerId)
+            .then((response) => setInventory(response.data))
+            .catch((error) => console.error(error));
+    }
+
+    function addCategory(category: string) {
+        axios.put(`/api/inventory/${inventory?.id}`, category, {headers:
```

Comment:
Is it wanted, that this can be run with inventory being undefined? `/api/inventory/` is not a valid entdpoint. You might want to put an if arount the axios command.